package.json

```diff
@@ -78,6 +78,7 @@
     "babel-plugin-transform-async-to-generator": "^6.24.1",
     "babel-plugin-transform-object-assign": "^6.22.0",
     "babel-plugin-transform-object-rest-spread": "^6.23.0",
+    "babel-plugin-transform-class-properties": "^6.24.1",
```

Comment:
Since this is a dev-dep of the data picker package, do we need it here?

Reply:
with out this here, the Travis CI breaks.